logstash-core/src/test/java/org/logstash/config/ir/compiler/DatasetCompilerTest.java

```diff
@@ -24,7 +25,7 @@ public void compilesOutputDataset() {
             DatasetCompiler.outputDataset(
                 Collections.emptyList(),
                 PipelineTestUtil.buildOutput(events -> {}),
-                true
```

Comment:
nitpick: previous was one line per argument, so adding a second argument to the last line feels _off_ to me.